r-pkg/R/get_fields.R

```diff
@@ -27,9 +27,10 @@ get_fields <- function(es_host
     # Input checking
     es_url <- .ValidateAndFormatHost(es_host)
 
+    # other input checks with simple error messages
     .assert(
-        is.character("es_indices")
```

Comment:
look look, this one was a bug!!!

It was always checking the literal string `"es_indices"` instead of checking the content of the argument `es_indices` 😅